docs/content/en/docs-dev/contribution-guidelines/development.md

```diff
@@ -11,6 +11,7 @@ description: >
 - [Go 1.19](https://go.dev/)
 - [Docker Decktop](https://www.docker.com/products/docker-desktop/)
 - [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) (If you want to run Control Plane locally)
+- [helm 3.8.2](https://helm.sh/docs/intro/install/) (If you want to run Control Plane locally)
```

Comment:
How about this?
```suggestion
- [helm 3.8](https://helm.sh/docs/intro/install/) (If you want to run Control Plane locally)
```

Reply:
The latest version of helm (3.10.2) is not supported, so helm 3.8 is used.